### ISSUE TITLE
Eduapage-api doesn't detect news

### ISSUE THREAD

**Issue by KexoTV** [ivanhrabcak/edupage-api#27]
```python
    news = edupage.get_news()
    channel_edu = client.get_channel(766574810376568832)
    print(news)
    for message in news:
        await channel_edu.send(str(message))
        print(str(message))
```
homework system works but news sadly returns blank list, 
![image](https://user-images.githubusercontent.com/55331891/132631559-3c68a3ac-d78d-426f-92d4-6ff7aecd4f7a.png)
does it detect news that are read?
```
**Version**
- Edupage API version: **(e.g. 0.9.81)**
- Python version: **(e.g. 3.9.7)**
```
jejda ja som idiot ja som to vypisoval po anglicky

**Comment by ivanhrabcak** [comment]
Ahoj, píš prosím po anglicky, tieto thready často čítajú aj iný (anglicky hovoriaci) ľudia, ktorý majú rovnaký problém. 
Ďakujem.

The problem could be, that the type of notification you are trying to fetch, is not news. News are only the notifications that have this icon: ![image](https://user-images.githubusercontent.com/39710116/132694169-11221a01-4487-4af6-872c-73462cd03825.jpeg)

You can check out `Edupage.get_notifications() `, maybe that's what you are looking for.

Please reply if this was the issue, or there is an issue with news fetching.

**Comment by KexoTV** [comment]
Hi, it works but i cannot get description of notification, it shows this string
`
("NotificationType.MESSAGE, Dôležitá správa, pre zobrazenie obsahu ju prosím otvorte [Prosím stiahnite si najnovšiu verziu mobilnej aplikácie]")
`

**Comment by ivanhrabcak** [comment]
Unfortunately I cannot reproduce this issue, but it looks like it is an issue with the code. If you want to, we can hop on a call on [this discord server](https://discord.gg/eU5eTenvNJ), and find out why this is happening.

**Comment by ivanhrabcak** [comment]
Closing as this issue was fixed in [0.9.82](https://github.com/ivanhrabcak/edupage-api/releases/tag/0.9.82).
